diode-server/README.md

```diff
@@ -62,11 +62,11 @@ curl -o .env https://raw.githubusercontent.com/netboxlabs/diode/release/diode-se
 Edit the `.env` to match your environment:
 
 * `NETBOX_DIODE_PLUGIN_API_BASE_URL`: URL for the Diode NetBox plugin API
-  least 40 characters, example generation with shell command: `openssl rand -base64 40 | head -c 40`)
-* `DIODE_TO_NETBOX_API_KEY`: API key generated with the Diode NetBox plugin installation
 * `MIGRATION_ENABLED`: Set to `false` to disable the migration, default: `true`
 * `RECONCILER_RATE_LIMITER_RPS`: Rate limit for the reconciler service for generating and applying change sets concurrently, default: `20`
 * `RECONCILER_RATE_LIMITER_BURST`: Burst limit for the reconciler service for generating and applying change sets concurrently, default: `1`
+* `DIODE_CLIENT_ID`: The OAuth2 Client ID to authenticate with
+* `DIODE_CLIENT_SECRET`: The OAuth2 Client secret to authenticate with
```

Comment:
Any thoughts of using `DIODE_CLIENT_*` here? I think it's too generic as it's not known for what direction of communication is it about. In the SDKs we use the same.

Reply:
hmm agree that it is confusing, also, a good behavior here would be use different ClientID and secret for this auth and SDK auth. With same var name, user may think that them should use same credential as the one in SDK